.github/workflows/cypress.yml

```diff
@@ -0,0 +1,42 @@
+name: E2E test
+
+on:
+  push:
+    branches:
+      - develop
+      - trunk
+  pull_request:
+    branches:
+      - develop
+jobs:
+  cypress:
+    name: ${{ matrix.core.name }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        core:
+          - {name: 'WP latest', version: 'latest'}
+          - {name: 'WP minimum', version: 'WordPress/WordPress#5.4'}
+          - {name: 'WP trunk', version: 'WordPress/WordPress#master'}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Install dependencies
+      run: npm install
+    - name: Composer (optional)
+      run: composer install
+      continue-on-error: true
+    - name: Build (optional)
+      run: npm run build
+      continue-on-error: true
+    - name: Set the core version
+      run: ./tests/bin/set-core-version.js ${{ matrix.core.version }}
+    - name: Set up WP environment
+      run: npm run env:start
+    - name: Set permissions
+      run: ./tests/bin/initialize.sh
```

Comment:
As we have `postenv:start` script, I believe this is already running after env:start.